services/cd-service/pkg/notify/notify.go

```diff
@@ -53,3 +56,31 @@ func (n *Notify) Notify() {
 		}
 	}
 }
+
+func (n *Notify) SubscribeChangesApps() (<-chan []string, Unsubscribe) {
```

Comment:
please also rename the other subscribe function to something like "SubscribeOverview".
That can be done in another PR if you prefer.